apps/docs/src/guide/creating-a-fuel-dapp/index.md

```diff
@@ -0,0 +1,179 @@
+# Creating a Fuel dApp
+
+`npm create fuels` is a command line tool that helps you scaffold a new fullstack Fuel dApp. In this guide, we will create a new counter dApp using `npm create fuels` and add decrement functionality to it. The final result will look like this:
```

Comment:
```suggestion
`create fuels` is a command line tool that helps you scaffold a new fullstack Fuel dApp. In this guide, we will create a new counter dApp using `create fuels` and add decrement functionality to it. The final result will look like this:
```

Is the package manager relevant in this case?

Reply:
I can't say yes or no. I just want the _brand_ of this package to be `npm create fuels` because the name itself is the command. If we only keep it `create-fuels` or `create fuels` then that's gonna be separate from the command, if you know what I mean?

Reply:
That does indeed make sense - it's all about the brand!